src/main/java/skynet/RobotWarSimulation.java

```diff
@@ -0,0 +1,45 @@
+package skynet;
+
+import java.util.Random;
+
+
+/**
+ * Main simulation class for the Robot War.
+ * Simulates 100 days of part production and collection
+ * and determines which faction builds the most robots.
+ */
+public class RobotWarSimulation {
+    /**
+     * Runs the Robot War simulation.
+     * Factory produces parts, factions collect them,
+     * and the winner is determined by the number of complete robots built.
+     *
+     * @param args command-line arguments (not used)
+     */
+    public static void main(String[] args) {
+        Factory factory = new Factory();
+        Faction world = new Faction("World", factory, 5);
+        Faction wednesday = new Faction("Wednesday", factory, 5);
+        Random random = new Random();
+
+        for (int day = 1; day <= 100; day++) {
```

Comment:
The simulation should have distinct day and night phases.  The current implementation doesn't separate these phases properly. There's no mechanism to ensure the factory produces parts during the day and factions collect at night in a coordinated manner.